src/odemis/acq/align/coordinates.py

```diff
@@ -324,7 +325,7 @@ def MatchCoordinates(input_coordinates, electron_coordinates, guess_scale, max_a
     # The ordered list gives for each electron coordinate the corresponding optical coordinates
     ordered_coordinates_index = sorted(zip(index1, electron_coordinates))
     ordered_coordinates = []
-    for i in xrange(len(ordered_coordinates_index)):
+    for i in range(len(ordered_coordinates_index)):
```

Comment:
Let's just write:
`ordered_coordinates = [ci[1] for ci in ordered_coordinates_index]`
Or maybe even fold it into the previous line into:
`ordered_coordinates = [ec for _, ec in sorted(zip(index1, electron_coordinates))]`